Vinyl/Composition/FuncComposition.swift

```diff
@@ -8,9 +8,14 @@
 
 import Foundation
 
-infix operator ~> { associativity left }
+precedencegroup VinylPrecedenceLeft {
+    associativity: left
+    higherThan: AssignmentPrecedence
```

Comment:
Unsure if this is the correct Precedence to use. As it's only used internally, maybe not an issue.